src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -168,10 +327,84 @@ export default class ManualBusinessInfo extends Vue {
     this.active = false
   }
 
+  /** When Jurisdiction value changes */
+  onJurisdictionChange (jurisdiction: any): void {
+    this.jurisdiction = jurisdiction
+    this.isMrasJurisdiction = MrasJurisdictions.includes(
+      this.jurisdiction?.text.toLowerCase()
+    )
+    if (jurisdiction?.group === 0) {
+      this.business.homeJurisdiction.country = JurisdictionLocation.CA
+      if (jurisdiction.value === JurisdictionLocation.FD) {
+        this.business.homeJurisdiction.region = 'Federal'
```

Comment:
I think this has to be all-caps.

https://github.com/bcgov/business-schemas/blob/92d132ebceabefc2ea3d453d596743081b076b1e/src/registry_schemas/schemas/foreign_jurisdiction.json#L22

Reply:
I think line 317 needs to be updated, too.